snippets/join_array_like_objects.md

```diff
@@ -0,0 +1,11 @@
+### Joining an array-like object
```

Comment:
I fixed the other two to comply with the styleguide and work properly, this one I cannot salvage myself. Can you provide an explanation of what you are trying to do? Otherwise delete this specific file and let's merge the other two who are solid. Thanks a lot for the contribution!